sonar-plugin-api/src/main/java/org/sonar/api/batch/PostJob.java

```diff
@@ -29,7 +29,9 @@
  * it is an asynchronous processing on server side.
  *
  * @since 1.10
+ * @deprecated since 5.6 use {@link org.sonar.api.batch.postjob.PostJob}
  */
+@Deprecated
 @BatchSide
 @ExtensionPoint
 public interface PostJob {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)